app/components/workflow-toggle.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Translate from 'react-translate-component';
+
+const WorkflowToggle = ({ workflow, handleToggle, name, checked }) => {
+  return (
+    <span>
+      { workflow.id } - { workflow.display_name}:
+      <label>
+        <input
+          type="checkbox"
+          name={name}
+          value={checked}
```

Comment:
[Checkbox input values should be strings](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox)

Reply:
Ok to go with "checked" as the string for input value here?

Reply:
Check out the example in the MDN article, along with the explanation: 
```
  <div>
    <input type="checkbox" id="subscribeNews" name="subscribe" value="newsletter">
    <label for="subscribeNews">Subscribe to newsletter?</label>
  </div>
```

> A DOMString representing the value of the checkbox. This is never seen on the client-side, but on the server this is the value given to the data submitted with the checkbox's name. In this example, we've got a name of subscribe, and a value of newsletter. When the form is submitted, the data name/value pair will be subscribe=newsletter. If the value attribute was omitted, the submitted data would be given a default value of on, so the submitted data in that case would be subscribe=on.

However, we're not using the normal browser behavior (no form is actually being submitted). So the value could be a string that makes sense for what the checkbox is toggling or it could be left off entirely. You can decide which makes the most sense in this case. 

I don't think the original component got this quite right either, so it's good to be mindful of errors in the old code.

Reply:
Cool, thanks. I was thinking it wasn't necessary, but wasn't sure if it was included originally because of best practices, or for some other reason. I'll remove it, since that seems like the cleanest solution.